api/docker-compose.yaml

```diff
@@ -20,165 +26,52 @@ services:
     ports:
       - 127.0.0.1:3306:3306
   api:
+    <<: *api-defaults
```

Comment:
I'm a bit surprised you need to use an anchor. I would've expected `docker-compose` to have a high-level concept for this kind of reuse.